doc/manual/manual.tex

```diff
@@ -5303,6 +5303,135 @@ \subsubsection{Simple convection in a spherical 3d shell}
 reality.
 
 
+\subsubsection{3D convection with an Earth-like initial condition}
+\label{sec:cookbooks-S20RTS}
+\textit{This section was contributed by Jacqueline Austermann}
+
+For any model run with \aspect{} we have to choose an initial condition for the 
+temperature field. If we want to model convection in the Earth's mantle we want 
+to choose an initial temperature distribution that captures the Earth's buoyancy 
+structure. In this cookbook we present how to use temperature perturbations 
+based on the shear wave velocity model S20RTS (\cite{S20RTS}) to 
+initialize a mantle convection calculation.
+
+\paragraph{The input shear wave model.}
+
+The current version of \aspect{} can read in the shear wave velocity models 
+S20RTS (\cite{S20RTS}) and S40RTS (\cite{S40RTS}), which are located 
+in \texttt{data/initial-conditions/S40RTS/}. Those models provide 
+spherical harmonic coefficients up do degree 20 and 40, respectively, for 21 
+depth layers. The interpolation with depth is done through a cubic spline 
+interpolation. The input files S20RTS.sph and S40RTS.sph were
+downloaded from \url{http://www.earth.lsa.umich.edu/~jritsema/Research.html} 
+and have the following format (this example is S20RTS): \\
+
+20 111111111111111111111  24 000111111111111111111111 \\
+  0.1534E-01 \\
+  0.1590E-01 -0.1336E-01  0.3469E-02 \\
+ -0.3480E-02  0.1165E-01  0.8376E-02  0.2158E-01 -0.9923E-02 \\
+... \\
+
+The first number in the first line denotes the maximum degree. This is followed in 
+the next line by the spherical harmonic coefficients from the surface down to the 
+CMB. The coefficients are arranged in the following way:\\
+
+\noindent $a_{00}$ \\
+$a_{10}$ $a_{11}$ $b_{11}$ \\
+$a_{20}$ $a_{21}$ $b_{21}$ $a_{22}$ $b_{22}$ \\
+... \\
+
+$a_{yz}$ is the cosine coefficient of degree $y$ and order $z$ and $b_{yz}$ is 
+the sine coefficient of degree $y$ and order $z$. The depth layers are specified 
+in the file Spline$\_$knots.txt by a normalized depth value ranging from the CMB (3480km, 
+normalized to -1) to the moho (6346km, normalized to 1). This is the original
+format provided on the homepage. 
+
+Any other perturbation model in this same format can also be used, one only
+has to specify the different filename in the parameter file (see next section).
+For models with different depth layers one has to adjust the Spline$\_$knots.txt 
```

Comment:
also, I don't think you need to pack `\_` between dollar signs, do you? I've never done it that way, I don't think.